services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py

```diff
@@ -176,6 +176,10 @@ async def store_to_s3(  # pylint:disable=too-many-locals,too-many-arguments
         f"{s3_retries}",
         "--transfers",
         f"{s3_parallelism}",
+        # limiting memory usage https://forum.rclone.org/t/how-to-set-a-memory-limit/10230/4
+        "--use-mmap",
```

Comment:
Could you comment on what these values are doing?

Reply:
They are telling go to use memory differently, this helps with freeing it faster.

Reply:
ok this is still not very helpful. what means differently?
Maybe just add comments in there as to there description and reference to the correct rclone documentation page.

Reply:
Let's see if now it looks better